it/pom.xml

```diff
@@ -88,41 +88,8 @@
   <profiles>
     <profile>
       <id>download-qa-artifacts</id>
```

Comment:
What is gonna activate the profile now? Shouldn't you keep an activation based on the presence of env.PROJECT_VERSION?
Is it still possible to run the ITs locally?

Reply:
Good point, I gave for granted that everything was working as long as the Env variable was defined locally but I did not tested it properly.
Let me have a look.

Reply:
Activation added back with the updated property PROJECT_VERSION